lms/djangoapps/teams/static/teams/js/spec/edit_team_spec.js

```diff
@@ -0,0 +1,189 @@
+            var descriptionFieldMessage = "A short description of the team to help other learners understand " +
+                "the goals or direction of the team (maximum 300 characters).";
+            expectContent('.u-field-description-message', descriptionFieldMessage);
+            expectContent('.u-field-optional_description .u-field-title', "Optional Characteristics");
+            var optionalDescriptionMessage = "Help other learners decide whether to join your team by specifying " +
+                "some characteristics for your team. Choose carefully, because fewer people might be interested in " +
+                "joining your team if it seems too restrictive.";
+            expectContent('.u-field-optional_description .u-field-message-help', optionalDescriptionMessage);
+            expectContent('.u-field-language .u-field-title', "Language");
+            expectContent(
+                '.u-field-language .u-field-message-help',
+                "The language that team members primarily use to communicate with each other."
+            );
+            expectContent('.u-field-country .u-field-title', "Country");
+            expectContent(
+                '.u-field-country .u-field-message-help', "The country that team members primarily identify with."
+            );
+
+            verifyDropdownData('.u-field-language select', [['a', 'aaa'], ['b', 'bbb']]);
+            verifyDropdownData('.u-field-country select', [['c', 'ccc'], ['d', 'ddd']]);
+
+            expect(teamEditView.$('.create-team.form-actions .action-primary').length).toBe(1);
+            expect(teamEditView.$('.create-team.form-actions .action-cancel').length).toBe(1);
+        });
+
+        it('can create a team', function () {
+            var requests = AjaxHelpers.requests(this);
+
+            teamEditView.$('.u-field-name input').val('TeamName');
+            teamEditView.$('.u-field-textarea textarea').val('TeamDescription');
+            teamEditView.$('.u-field-language select').val('a').attr("selected", "selected");
+            teamEditView.$('.u-field-country select').val('c').attr("selected", "selected");
+
+            teamEditView.$('.create-team.form-actions .action-primary').click();
+            AjaxHelpers.expectJsonRequest(requests, 'POST', teamsUrl, teamsData);
+            AjaxHelpers.respondWithJson(requests, teamsData);
+
+            expect(teamEditView.$('.create-team.wrapper-msg .copy').text().trim().length).toBe(0);
+            expect(Backbone.history.navigate.calls[0].args).toContain('topics/awesomeness');
+        });
+
+        it('shows validation error message when field is empty', function () {
+            var requests = AjaxHelpers.requests(this);
+            verifyValidation(requests, [
+                ['.u-field-name input', 'Name', 'success'],
+                ['.u-field-textarea textarea', '', 'error']
+            ]);
+            teamEditView.render();
```

Comment:
I think we have discussed to render the view between these verifyValidation calls.